examples/demo_coordinate_system_datasets.ipynb

```diff
@@ -0,0 +1,916 @@
+    "redshifts=np.ones(np.shape(data)[0])\n",
+    "galaxies = clmm.GCData(\n",
+    "    [ra, dec, e1, e2, redshifts, ids], names=[\"ra\", \"dec\", \"e1\", \"e2\", \"z\", \"id\"]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here we create a GalaxyCluster object, specifying the correct coordinate system. For source galaxies from the Oguri curated dataset, these are in the euclidean coordinate system. We use the implemented kwarg when defining the galaxy cluster object to also **specify the euclidean coordinate system**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster = clmm.GalaxyCluster(\"Name of cluster\", ra_cl, dec_cl, zl, galaxies, coordinate_system=\"euclidean\")\n",
+    "\n",
+    "# Convert elipticities into shears for the members.\n",
+    "cluster.compute_tangential_and_cross_components()\n",
+    "print(cluster.galcat.colnames)\n",
+    "\n",
+    "# Measure profile and add profile table to the cluster.\n",
+    "seps = convert_units(cluster.galcat[\"theta\"], \"radians\", \"Mpc\", cluster.z, cosmo)\n",
+    "\n",
+    "cluster.make_radial_profile(\n",
+    "    bins=da.make_bins(0.2, 3.0, 8, method=\"evenlog10width\"),\n",
+    "    bin_units=\"Mpc\",\n",
+    "    cosmo=cosmo,\n",
+    "    include_empty_bins=False,\n",
+    "    gal_ids_in_bins=True,\n",
+    ")\n",
+    "print(cluster.profile.colnames)\n",
+    "\n",
+    "\n",
+    "fig = plt.figure(figsize=(6, 4))\n",
+    "ax = fig.add_axes((0, 0, 1, 1))\n",
+    "errorbar_kwargs = dict(linestyle=\"\", marker=\"o\", markersize=1, elinewidth=0.5, capthick=0.5)\n",
+    "ax.errorbar(\n",
+    "    cluster.profile[\"radius\"],\n",
+    "    cluster.profile[\"gt\"],\n",
+    "    cluster.profile[\"gt_err\"],\n",
+    "    label='euclidean'\n",
+    ")\n",
+    "\n",
+    "\n",
+    "cluster.compute_tangential_and_cross_components()\n",
+    "print(cluster.galcat.colnames)\n",
+    "\n",
+    "# Measure profile and add profile table to the cluster.\n",
+    "seps = convert_units(cluster.galcat[\"theta\"], \"radians\", \"Mpc\", cluster.z, cosmo)\n",
+    "\n",
+    "cluster.make_radial_profile(\n",
+    "    bins=da.make_bins(0.1, 3.0, 8, method=\"evenlog10width\"),\n",
+    "    bin_units=\"Mpc\",\n",
+    "    cosmo=cosmo,\n",
+    "    include_empty_bins=False,\n",
+    "    gal_ids_in_bins=True,\n",
+    ")\n",
+    "print(cluster.profile.colnames)\n",
+    "\n",
+    "ax.errorbar(\n",
+    "    cluster.profile[\"radius\"],\n",
+    "    cluster.profile[\"gt\"],\n",
+    "    cluster.profile[\"gt_err\"],\n",
+    "    label='celestial'\n",
+    ")\n",
+    "\n",
+    "\n",
+    "ax.set_xlabel(\"r [Mpc]\", fontsize=20)\n",
+    "ax.set_ylabel(r\"$g_t$\", fontsize=20)\n",
+    "ax.grid(lw=0.3)\n",
+    "ax.minorticks_on()\n",
+    "ax.grid(which=\"minor\", lw=0.1)\n",
+    "plt.legend(fontsize=20)\n",
+    "plt.show()"
```

Comment:
Here you are plotting the same data, only in different bins. One of them should be in celestial coordinates.

Reply:
should be now addressed in latest push.

Reply:
(I'm about to test the notebook on NERSC)